week-1-practise/level-1/8-function-output-1.js

```diff
@@ -1,12 +1,12 @@
 function getNumber() {
-    return Math.random() * 10;
+  return Math.random() * 10;
 }
 
 var result = getNumber();
 console.log(result);
 
 // Answer these questions:
-// - What does this program do?
-// - How many parameters does the function take?
-// - What is the function name?
-// - Where is the function called? with what parameters?
\ No newline at end of file
+// - What does this program do? first pick up a number between 0 and 1 randomly and after multiply it with 10.
+// - How many parameters does the function take? this function doesn`t need any parameters.
+// - What is the function name? function name is getNumber().
```

Comment:
Very close but not quite. It's just a subtlety but I suggest reading the first few paragraphs here: http://2ality.com/2015/09/function-names-es6.html.

Reply:
You've got it right in `9-function-output-2.js` 😀